remote-access/ssh/README.md

```diff
@@ -1,14 +1,33 @@
 # SSH (Secure Shell)
 
-You can remotely gain access to the command line of a Raspberry Pi from another computer on the same network using `ssh`.
+You can access the command line of a Raspberry Pi remotely from another computer or device on the same network using `ssh`.
 
-Note you only have access to the command line, not the full desktop environment. For full remote desktop see [VNC](../vnc/README.md).
+##1. Setup your local network and Wi-Fi
+Make sure your Raspberry Pi is properly set up and connected. If you're using [Wi-Fi](https://www.raspberrypi.org/documentation/configuration/wireless/wireless-cli.md), this is done using the `wpa_supplicant.conf` config file. 
 
-You can enable or disable the SSH server on your Raspberry Pi (it is enabled by default). This is done using [raspi-config](../../configuration/raspi-config.md):
+Add the following data to the file:
+```
+network={
+    ssid="Your_SSID"
+    psk="Your_wifi_password"
+}
```

Comment:
This is duplicating information from another page, and so probably shouldn't be included here?